commons/protocol/src/test/resource/saved-protocol/0.1/complex/build_structure.json

```diff
@@ -1 +1 @@
-{"builds":["file:///test/project"],"projects":[{"id":{"build":"file:///test/project","name":"test"},"plugins":["com.foo.Plugin"]}]}
\ No newline at end of file
+{"$type":"sbt.protocol.MinimalBuildStructure","projects":[{"$type":"sbt.protocol.MinimalProjectStructure","plugins":["com.foo.Plugin"],"id":{"name":"test","build":"file:///test/project"}}],"builds":["file:///test/project"]}
```

Comment:
Still not sure why the type isn't elided.  I suspect dynamic pickling is still getting used somewhere.